core/src/main/scala/org/apache/spark/scheduler/cluster/CoarseGrainedSchedulerBackend.scala

```diff
@@ -176,11 +182,21 @@ class CoarseGrainedSchedulerBackend(scheduler: TaskSchedulerImpl, val rpcEnv: Rp
         }
 
       case KillExecutorsOnHost(host) =>
-        scheduler.getExecutorsAliveOnHost(host).foreach { exec =>
-          killExecutors(exec.toSeq, adjustTargetNumExecutors = false, countFailures = false,
+        scheduler.getExecutorsAliveOnHost(host).foreach { execs =>
+          killExecutors(execs.toSeq, adjustTargetNumExecutors = false, countFailures = false,
```

Comment:
If you don't mind, shall we revert this change to reduce this patch size?

Reply:
So I switched the name to `execs` since it is multiple executors and I was having difficulty reading the code the first time through. It's not strong feelings but I think it's clearer this way (e.g. it's not calling foreach on the executors it's actually calling foreach on the option of list of execs).

Reply:
Got it. It's up to your decision.

Reply:
cool, if your ok with it I find this makes the code more readable so I'll leave as is :)